confignode/src/main/java/org/apache/iotdb/confignode/persistence/NodeInfo.java

```diff
@@ -112,7 +112,9 @@ public void addMetrics() {
               registeredConfigNodes,
               o -> getRegisteredConfigNodeCount(),
               Tag.NAME.toString(),
-              "online");
+              "total",
+              Tag.STATUS.toString(),
+              "registered");
```

Comment:
I think the value of tag can be the name of NodeStatus enums to remove hard code.